packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-cases/oie-embedded-widget-use-case-load/go/integrationsteps.md

```diff
@@ -0,0 +1,130 @@
+### Step 1: Click on the sign-in page link
+
+The first step occurs when the user clicks on the sign-in link. This link
+points to the sign-in page where the Widget is embedded. On the sample application's homepage,
+this link is named **Login** and located on the top right corner of the page.
+
+### Step 2: Get Widget initialization data
+
+When the sign-in page loads, obtain the parameters required for the Widget to load. Source these
+parameters using multiple methods. The main parameters include:
+
+* **Client ID, issuer, scopes:** source from the [configuration](/docs/guides/oie-embedded-common-download-setup-app/go/main/#configuration-settings)
+* **Interaction Handle**: obtain from the `/interact` endpoint
+* **PCKE parameters, state and nonce:** generate values
+* **Base URL:** derive from issuer URL
+
+These parameters are passed to the Widget as it loads on the page. Where you set these
+values depends on the application. The sample application sets most of these values on the server
```

Comment:
What dependency does the setting location have with the application?

Reply:
I felt the sentences were not necessary as they are explaining details that developers should know. They were removed.